### DISCUSSION TITLE
What is Debug Depth for?

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#186] [Q&A] [answered]
What is Debug Depth for? I see a video that has two different depth maps. There's some numbers upper left corner ( min=,max=, mean=, std=) What do they represent? I see the 'resolution' is the same as what would get exported with export disparity (x2 width). Is the right side the same as 'Export Disparity' depth map? What is the difference between the two sides and how would it be useful to understand? Thanks.

![Debug Depth 001](https://github.com/user-attachments/assets/bbcd50c6-be4c-4273-a8c4-dcf19d964c74)

**Comment by nagadomi** [comment, marked as answer]
It is a feature that just visualizes the output of the depth model and the mapper function (foreground-scale).
It is only for my development purposes and is not intended for user use.
The image on the left is the raw depth output and the image on the right is it converted to disparity for stereo generation.

**Reply by diverswan9** [reply]
Thanks... I'll walk around it then.

**Comment by tufeixp** [comment]
Please add an option for saving sbs rgbd video in full width, I can use that on my glasses-free monitor, thanks!

**Reply by nagadomi** [reply]
Please open a new issue.
I'm not familiar with what the standard RGBD video formats are (I know of a few products that support it, but I'm not certain whether they all use the same standard)
so if you have a link to the format specifications supported by that product, please include it.
